beamer/chain.py

```diff
@@ -405,13 +405,20 @@ def maybe_withdraw(claim: Claim, context: Context) -> None:
     # This gives us a chronological order. The agent should never garbage collect
     # a request which has active claims
     assert request is not None, "Active claim for non-existent request"
+    assert context.address in {claim.claimer, claim.challenger}, "Agent is not part of claim"
```

Comment:
I think we have to remove that line with the multi party bidding scheme anyway.

Reply:
Already removed this as it caused problems in the tests.